l10n_br_account/models/account_invoice.py

```diff
@@ -43,6 +43,7 @@
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
+    _order = "internal_number desc, id desc"
```

Comment:
We should find a better way to sort this, as internal_number is a char field, this does not work as expected.
![broken order](https://cloud.githubusercontent.com/assets/854781/16233066/6901b69e-37a3-11e6-84a3-5d9717a80afe.png)